corehq/apps/app_manager/templates/app_manager/v1/summary.html

```diff
@@ -36,6 +36,7 @@
                 $httpProvider.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
                 $httpProvider.defaults.xsrfCookieName = 'csrftoken';
                 $httpProvider.defaults.xsrfHeaderName = 'X-CSRFToken';
+                $httpProvider.defaults.headers.common["X-CSRFToken"] = '{{ csrf_token }}';
```

Comment:
@orangejenny / @biyeun Can you help understand what i am missing here? I have done the change for v2 summary as well and using `build_app_manager_v2_diffs` generates no diff files. But the build fails: https://travis-ci.org/dimagi/commcare-hq/jobs/215294313